service/iot-service-client/src/main/java/com/microsoft/azure/sdk/iot/service/devicetwin/DeviceMethodClientOptions.java

```diff
@@ -0,0 +1,39 @@
+package com.microsoft.azure.sdk.iot.service.devicetwin;
+
+import com.microsoft.azure.sdk.iot.service.ProxyOptions;
+import lombok.Builder;
+import lombok.Getter;
+
+/**
+ * Configurable options for all device method operations.
+ */
+@Builder
+public class DeviceMethodClientOptions
+{
+    protected static final Integer DEFAULT_HTTP_READ_TIMEOUT_MS = 24000; // 24 seconds
+    protected static final Integer DEFAULT_HTTP_CONNECT_TIMEOUT_MS = 24000; // 24 seconds
```

Comment:
Why are these values 24?

Reply:
These were the defaults before my changes here